onnxruntime/python/providers/tvm/ort.py

```diff
@@ -53,6 +32,26 @@ def onnx_compile(model_string,
                  nhwc=False,
                  tuning_logfile="",
                  tuning_type=AUTO_TVM_TYPE):
+    def get_tvm_executor(irmod, executor, target, target_host, params):
+        # TODO(vvchernov): replace prints by logger, but investigate ORT logging system for python before
+        # see lines 69, 77, 119, 136
+        if executor == "vm":
+            # print("Build TVM virtual machine")
+            lib = vm.compile(
+                copy.deepcopy(irmod),
+                target,
+                params=params,
+                target_host=target_host,
+            )
+        elif executor == "graph":
+            # print("Build TVM graph executor")
+            lib = relay.build(irmod, target=target, target_host=target_host, params=params)
+        else:
+            print("ERROR: Executor type {} is unsupported. ".format(executor),
```

Comment:
No exception?

Reply:
I'm not sure that exception is caught correctly in python part launched from native code. I added error logging in python and check with error of nullptr output on native side